projboard/tests/test_user.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from projboard.models.user import User
+
+
+NAME = "RAWAD"
+NICKNAME = "R1998"
+EMAIL = "rawad@gmail.com"
+PASSWORD = "rawad!@#"
+
+
+@pytest.fixture
+def generate_user():
+    user = User(name=NAME, nickname=NICKNAME, email=EMAIL, password=PASSWORD)
+    user.save()
+    return user
+
+
+class TestUserModel:
+    @pytest.fixture
+    def test_new_user(self, generate_user):
+        assert generate_user.name == NAME
+        assert generate_user.nickname == NICKNAME
+        assert generate_user.email == EMAIL
+        assert generate_user.password == PASSWORD
```

Comment:
I assume you should use this 
User.get_user(generate_user.email,  generate_user.nickname, generate_user.name, generate_user.nickname):

And then assert it?

Reply:
@Yarboa Done

Reply:
But you have to assert the User content with user_generated data,
You should compare what the test prepared with what was generated in DB
Dont you think?

Reply:
@Yarboa Done